lms/static/js/accessible_components/StatusBarAlert.jsx

```diff
@@ -0,0 +1,43 @@
+/*
+Wrapper for React/Paragon accessible status bar
+*/
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import StatusAlert from '@edx/paragon/src/StatusAlert/index.jsx';
+
+export class StatusAlertRenderer {
+  constructor(message, selector, afterselector) {
+    this.shiftFocus = this.shiftFocus.bind(this);
+    const element = document.querySelector(selector);
+
+    if (element) {
+      /*
+      These props match the defaults mostly in the paragon lib:
+      https://github.com/edx/paragon/tree/master/src/StatusAlert
+      but are made explicit in the case of a upstream change to defaults
+      */
+      ReactDOM.render(
+          <StatusAlert
+              alertType='warning'
+              dismissible={true}
+              open={true}
+              dialog={message}
+              dismissable={true}
+              onClose={() => this.shiftFocus(afterselector)}
+          />,
+          document.querySelector(selector)
```

Comment:
Nit: can you use 2-space indentation for everything in this block? You can do this automatically with `--fix` locally (within edx-platform root dir):
```
./node_modules/.bin/eslint --ext .jsx lms/static/js/accessible_components/ --fix
```
Eslint *should* throw errors in our quality job, but it doesn't  currently run on jsx files [due to python tooling limitations](https://openedx.atlassian.net/browse/FEDX-374).

Reply:
👍